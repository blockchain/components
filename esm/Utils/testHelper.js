import React from 'react';
import { shallow, mount } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import theme from '../Tools/theme';

function wrapWithTheme(fn, children, options) {
  var wrapper = fn(React.createElement(ThemeProvider, {
    theme: theme
  }, children), options);
  return wrapper[fn.name]({
    context: wrapper.instance().getChildContext()
  });
}

export function shallowWithTheme() {
  return wrapWithTheme.apply(void 0, [shallow].concat(Array.prototype.slice.call(arguments)));
}
export function mountWithTheme() {
  return wrapWithTheme.apply(void 0, [mount].concat(Array.prototype.slice.call(arguments)));
}