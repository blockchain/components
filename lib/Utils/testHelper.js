"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.shallowWithTheme = shallowWithTheme;
exports.mountWithTheme = mountWithTheme;

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _styledComponents = require("styled-components");

var _theme = _interopRequireDefault(require("../Tools/theme"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function wrapWithTheme(fn, children, options) {
  var wrapper = fn(_react.default.createElement(_styledComponents.ThemeProvider, {
    theme: _theme.default
  }, children), options);
  return wrapper[fn.name]({
    context: wrapper.instance().getChildContext()
  });
}

function shallowWithTheme() {
  return wrapWithTheme.apply(void 0, [_enzyme.shallow].concat(Array.prototype.slice.call(arguments)));
}

function mountWithTheme() {
  return wrapWithTheme.apply(void 0, [_enzyme.mount].concat(Array.prototype.slice.call(arguments)));
}