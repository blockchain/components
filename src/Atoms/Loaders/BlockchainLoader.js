import React from 'react'
import PropTypes from 'prop-types'
import styled, { keyframes } from 'styled-components'

const scaleDelay = keyframes`
  0%,
  50%,
  100% { transform: scale3d(1, 1, 1); }
  25% { transform: scale3d(0, 0, 1); }
`
const Wrapper = styled.div`
  position: relative;
  width: ${(props) => props.size};
  height: ${(props) => props.size};
`
const Container = styled.div`
  width: 70%;
  height: 70%;
  transform-origin: 85% 35%;
  transform: rotate(-45deg);
  border-radius: 10%;
  overflow: hidden;
`
const Square = styled.div`
  display: inline-block;
  width: 20%;
  height: 20%;
  float: left;
  border: 0;
  animation: ${scaleDelay} 3s infinite ease-in-out;
`
const SquareA = styled(Square)`
  background-color: #153a62;
`
const SquareB = styled(Square)`
  background-color: #799eb2;
`
const SquareC = styled(Square)`
  background-color: #3558a8;
`
const SquareD = styled(Square)`
  background-color: #b2d5e5;
`
const SquareE = styled(Square)`
  background-color: #10ade4;
`
const Square1 = styled(SquareA)`
  animation-delay: 0s;
`
const Square2 = styled(SquareA)`
  animation-delay: 0.1s;
`
const Square3 = styled(SquareA)`
  animation-delay: 0.2s;
`
const Square4 = styled(SquareB)`
  animation-delay: 0.3s;
`
const Square5 = styled(SquareB)`
  animation-delay: 0.4s;
`
const Square6 = styled(SquareA)`
  animation-delay: 0.1s;
`
const Square7 = styled(SquareA)`
  animation-delay: 0.2s;
`
const Square8 = styled(SquareA)`
  animation-delay: 0.3s;
`
const Square9 = styled(SquareB)`
  animation-delay: 0.4s;
`
const Square10 = styled(SquareB)`
  animation-delay: 0.5s;
`
const Square11 = styled(SquareE)`
  animation-delay: 0.2s;
`
const Square12 = styled(SquareE)`
  animation-delay: 0.3s;
`
const Square13 = styled(SquareE)`
  animation-delay: 0.4s;
`
const Square14 = styled(SquareC)`
  animation-delay: 0.5s;
`
const Square15 = styled(SquareC)`
  animation-delay: 0.6s;
`
const Square16 = styled(SquareE)`
  animation-delay: 0.3s;
`
const Square17 = styled(SquareE)`
  animation-delay: 0.4s;
`
const Square18 = styled(SquareE)`
  animation-delay: 0.5s;
`
const Square19 = styled(SquareC)`
  animation-delay: 0.6s;
`
const Square20 = styled(SquareC)`
  animation-delay: 0.7s;
`
const Square21 = styled(SquareE)`
  animation-delay: 0.4s;
`
const Square22 = styled(SquareE)`
  animation-delay: 0.5s;
`
const Square23 = styled(SquareE)`
  animation-delay: 0.6s;
`
const Square24 = styled(SquareD)`
  animation-delay: 0.7s;
`
const Square25 = styled(SquareD)`
  animation-delay: 0.8s;
`

const BlockchainLoader = (props) => (
  <Wrapper {...props}>
    <Container>
      <Square1 />
      <Square2 />
      <Square3 />
      <Square4 />
      <Square5 />
      <Square6 />
      <Square7 />
      <Square8 />
      <Square9 />
      <Square10 />
      <Square11 />
      <Square12 />
      <Square13 />
      <Square14 />
      <Square15 />
      <Square16 />
      <Square17 />
      <Square18 />
      <Square19 />
      <Square20 />
      <Square21 />
      <Square22 />
      <Square23 />
      <Square24 />
      <Square25 />
    </Container>
  </Wrapper>
)

BlockchainLoader.propTypes = {
  size: PropTypes.string.isRequired,
}

BlockchainLoader.defaultProps = {
  size: '150px',
}

export default BlockchainLoader
