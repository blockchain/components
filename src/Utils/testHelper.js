import React from 'react'
import { shallow, mount } from 'enzyme'
import { ThemeProvider } from 'styled-components'
import theme from '../Tools/theme'

function wrapWithTheme(fn, children, options) {
  const wrapper = fn(
    <ThemeProvider theme={theme}>{children}</ThemeProvider>,
    options,
  )

  return wrapper[fn.name]({
    context: wrapper.instance().getChildContext(),
  })
}

export function shallowWithTheme() {
  return wrapWithTheme(shallow, ...arguments)
}

export function mountWithTheme() {
  return wrapWithTheme(mount, ...arguments)
}
