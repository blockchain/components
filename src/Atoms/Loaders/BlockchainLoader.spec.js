import React from 'react'
import { mountWithTheme } from '../../Utils/testHelper'

import BlockchainLoader from './BlockchainLoader'

describe('BlockchainLoader', () => {
  it('should render correctly', () => {
    const tree = mountWithTheme(<BlockchainLoader />)

    expect(tree.find(BlockchainLoader)).toMatchSnapshot()
  })

  it('should render correctly with size', () => {
    const tree = mountWithTheme(<BlockchainLoader size="200px" />)

    expect(tree.find(BlockchainLoader)).toMatchSnapshot()
  })
})
