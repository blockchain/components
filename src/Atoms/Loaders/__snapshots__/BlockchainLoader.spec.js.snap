// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockchainLoader should render correctly 1`] = `
.c0 {
  position: relative;
  width: 150px;
  height: 150px;
}

.c1 {
  width: 70%;
  height: 70%;
  -webkit-transform-origin: 85% 35%;
  -ms-transform-origin: 85% 35%;
  transform-origin: 85% 35%;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  border-radius: 10%;
  overflow: hidden;
}

.c4 {
  display: inline-block;
  width: 20%;
  height: 20%;
  float: left;
  border: 0;
  -webkit-animation: ijAREv 3s infinite ease-in-out;
  animation: ijAREv 3s infinite ease-in-out;
}

.c3 {
  background-color: #153a62;
}

.c8 {
  background-color: #799eb2;
}

.c20 {
  background-color: #3558a8;
}

.c31 {
  background-color: #b2d5e5;
}

.c16 {
  background-color: #10ade4;
}

.c2 {
  -webkit-animation-delay: 0s;
  animation-delay: 0s;
}

.c5 {
  -webkit-animation-delay: 0.1s;
  animation-delay: 0.1s;
}

.c6 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c7 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c9 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c10 {
  -webkit-animation-delay: 0.1s;
  animation-delay: 0.1s;
}

.c11 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c12 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c13 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c14 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c15 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c17 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c18 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c19 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c21 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c22 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c23 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c24 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c25 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c26 {
  -webkit-animation-delay: 0.7s;
  animation-delay: 0.7s;
}

.c27 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c28 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c29 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c30 {
  -webkit-animation-delay: 0.7s;
  animation-delay: 0.7s;
}

.c32 {
  -webkit-animation-delay: 0.8s;
  animation-delay: 0.8s;
}

<BlockchainLoader
  size="150px"
>
  <styled.div
    size="150px"
  >
    <div
      className="c0"
      size="150px"
    >
      <styled.div>
        <div
          className="c1"
        >
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c2"
            >
              <styled.div
                className="c2 c3"
              >
                <div
                  className="c2 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c5"
            >
              <styled.div
                className="c5 c3"
              >
                <div
                  className="c5 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c6"
            >
              <styled.div
                className="c6 c3"
              >
                <div
                  className="c6 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c7"
            >
              <styled.div
                className="c7 c8"
              >
                <div
                  className="c7 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c9"
            >
              <styled.div
                className="c9 c8"
              >
                <div
                  className="c9 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c10"
            >
              <styled.div
                className="c10 c3"
              >
                <div
                  className="c10 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c11"
            >
              <styled.div
                className="c11 c3"
              >
                <div
                  className="c11 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c12"
            >
              <styled.div
                className="c12 c3"
              >
                <div
                  className="c12 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c13"
            >
              <styled.div
                className="c13 c8"
              >
                <div
                  className="c13 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c14"
            >
              <styled.div
                className="c14 c8"
              >
                <div
                  className="c14 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c15"
            >
              <styled.div
                className="c15 c16"
              >
                <div
                  className="c15 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c17"
            >
              <styled.div
                className="c17 c16"
              >
                <div
                  className="c17 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c18"
            >
              <styled.div
                className="c18 c16"
              >
                <div
                  className="c18 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c19"
            >
              <styled.div
                className="c19 c20"
              >
                <div
                  className="c19 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c21"
            >
              <styled.div
                className="c21 c20"
              >
                <div
                  className="c21 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c22"
            >
              <styled.div
                className="c22 c16"
              >
                <div
                  className="c22 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c23"
            >
              <styled.div
                className="c23 c16"
              >
                <div
                  className="c23 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c24"
            >
              <styled.div
                className="c24 c16"
              >
                <div
                  className="c24 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c25"
            >
              <styled.div
                className="c25 c20"
              >
                <div
                  className="c25 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c26"
            >
              <styled.div
                className="c26 c20"
              >
                <div
                  className="c26 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c27"
            >
              <styled.div
                className="c27 c16"
              >
                <div
                  className="c27 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c28"
            >
              <styled.div
                className="c28 c16"
              >
                <div
                  className="c28 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c29"
            >
              <styled.div
                className="c29 c16"
              >
                <div
                  className="c29 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c30"
            >
              <styled.div
                className="c30 c31"
              >
                <div
                  className="c30 c31 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c32"
            >
              <styled.div
                className="c32 c31"
              >
                <div
                  className="c32 c31 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
        </div>
      </styled.div>
    </div>
  </styled.div>
</BlockchainLoader>
`;

exports[`BlockchainLoader should render correctly with size 1`] = `
.c0 {
  position: relative;
  width: 200px;
  height: 200px;
}

.c1 {
  width: 70%;
  height: 70%;
  -webkit-transform-origin: 85% 35%;
  -ms-transform-origin: 85% 35%;
  transform-origin: 85% 35%;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  border-radius: 10%;
  overflow: hidden;
}

.c4 {
  display: inline-block;
  width: 20%;
  height: 20%;
  float: left;
  border: 0;
  -webkit-animation: ijAREv 3s infinite ease-in-out;
  animation: ijAREv 3s infinite ease-in-out;
}

.c3 {
  background-color: #153a62;
}

.c8 {
  background-color: #799eb2;
}

.c20 {
  background-color: #3558a8;
}

.c31 {
  background-color: #b2d5e5;
}

.c16 {
  background-color: #10ade4;
}

.c2 {
  -webkit-animation-delay: 0s;
  animation-delay: 0s;
}

.c5 {
  -webkit-animation-delay: 0.1s;
  animation-delay: 0.1s;
}

.c6 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c7 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c9 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c10 {
  -webkit-animation-delay: 0.1s;
  animation-delay: 0.1s;
}

.c11 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c12 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c13 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c14 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c15 {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c17 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c18 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c19 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c21 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c22 {
  -webkit-animation-delay: 0.3s;
  animation-delay: 0.3s;
}

.c23 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c24 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c25 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c26 {
  -webkit-animation-delay: 0.7s;
  animation-delay: 0.7s;
}

.c27 {
  -webkit-animation-delay: 0.4s;
  animation-delay: 0.4s;
}

.c28 {
  -webkit-animation-delay: 0.5s;
  animation-delay: 0.5s;
}

.c29 {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c30 {
  -webkit-animation-delay: 0.7s;
  animation-delay: 0.7s;
}

.c32 {
  -webkit-animation-delay: 0.8s;
  animation-delay: 0.8s;
}

<BlockchainLoader
  size="200px"
>
  <styled.div
    size="200px"
  >
    <div
      className="c0"
      size="200px"
    >
      <styled.div>
        <div
          className="c1"
        >
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c2"
            >
              <styled.div
                className="c2 c3"
              >
                <div
                  className="c2 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c5"
            >
              <styled.div
                className="c5 c3"
              >
                <div
                  className="c5 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c6"
            >
              <styled.div
                className="c6 c3"
              >
                <div
                  className="c6 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c7"
            >
              <styled.div
                className="c7 c8"
              >
                <div
                  className="c7 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c9"
            >
              <styled.div
                className="c9 c8"
              >
                <div
                  className="c9 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c10"
            >
              <styled.div
                className="c10 c3"
              >
                <div
                  className="c10 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c11"
            >
              <styled.div
                className="c11 c3"
              >
                <div
                  className="c11 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c12"
            >
              <styled.div
                className="c12 c3"
              >
                <div
                  className="c12 c3 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c13"
            >
              <styled.div
                className="c13 c8"
              >
                <div
                  className="c13 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c14"
            >
              <styled.div
                className="c14 c8"
              >
                <div
                  className="c14 c8 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c15"
            >
              <styled.div
                className="c15 c16"
              >
                <div
                  className="c15 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c17"
            >
              <styled.div
                className="c17 c16"
              >
                <div
                  className="c17 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c18"
            >
              <styled.div
                className="c18 c16"
              >
                <div
                  className="c18 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c19"
            >
              <styled.div
                className="c19 c20"
              >
                <div
                  className="c19 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c21"
            >
              <styled.div
                className="c21 c20"
              >
                <div
                  className="c21 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c22"
            >
              <styled.div
                className="c22 c16"
              >
                <div
                  className="c22 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c23"
            >
              <styled.div
                className="c23 c16"
              >
                <div
                  className="c23 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c24"
            >
              <styled.div
                className="c24 c16"
              >
                <div
                  className="c24 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c25"
            >
              <styled.div
                className="c25 c20"
              >
                <div
                  className="c25 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c26"
            >
              <styled.div
                className="c26 c20"
              >
                <div
                  className="c26 c20 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c27"
            >
              <styled.div
                className="c27 c16"
              >
                <div
                  className="c27 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c28"
            >
              <styled.div
                className="c28 c16"
              >
                <div
                  className="c28 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c29"
            >
              <styled.div
                className="c29 c16"
              >
                <div
                  className="c29 c16 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c30"
            >
              <styled.div
                className="c30 c31"
              >
                <div
                  className="c30 c31 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
          <Styled(Styled(styled.div))>
            <Styled(styled.div)
              className="c32"
            >
              <styled.div
                className="c32 c31"
              >
                <div
                  className="c32 c31 c4"
                />
              </styled.div>
            </Styled(styled.div)>
          </Styled(Styled(styled.div))>
        </div>
      </styled.div>
    </div>
  </styled.div>
</BlockchainLoader>
`;
